config/seata-config-nacos/src/main/java/io/seata/config/nacos/NacosConfiguration.java

```diff
@@ -47,13 +47,31 @@
     private static volatile ConfigService configService;
 
     /**
-     * Instantiates a new Nacos configuration.
+     * Get instance of NacosConfiguration
      *
-     * @throws NacosException the nacos exception
+     * @return
+     */
+    public static NacosConfiguration getInstance() {
+        if (null == instance) {
+            synchronized (NacosConfiguration.class) {
+                if (null == instance) {
+                    instance = new NacosConfiguration();
+                }
+            }
+        }
+        return instance;
+    }
+
+    /**
+     * Instantiates a new Nacos configuration.
      */
-    public NacosConfiguration() throws NacosException {
+    public NacosConfiguration() {
         if (null == configService) {
-            configService = NacosFactory.createConfigService(getConfigProperties());
+            try {
+                configService = NacosFactory.createConfigService(getConfigProperties());
+            } catch (NacosException e) {
+                throw new RuntimeException(e.getMessage());
```

Comment:
throw e;

Reply:
throw e won't work here.because NacosException is not a subclass of RuntimeException.

Reply:
throw new RuntimeException(e);

Reply:
> throw new RuntimeException(e);

done